ci/infra/azure/cloud-init/repository.tpl

```diff
@@ -1 +1,3 @@
-zypper ar -f ${repository_url} ${repository_name}
+  - zypper addrepo --refresh ${repository_url} ${repository_name}
```

Comment:
It seems the formatting has some difference, extra indent in the beginning of each line?

Reply:
Yes. Before we vritual-machine-extenstion to run script and now we use cloud-init.